src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Equinor.ProCoSys.IPO.WebApi.IntegrationTests.Invitations
+{
+    [TestClass]
+    public class InvitationsControllerTests : InvitationsControllerTestsBase
+    {
+        [TestMethod]
+        public async Task GetInvitation_AsViewer_ShouldGetInvitation()
+        {
+            // Act
+            var invitation = await InvitationsControllerTestsHelper.GetInvitationAsync(
+                ViewerClient(TestFactory.PlantWithAccess), 
+                InitialInvitationId);
+
+            // Assert
+            Assert.IsNotNull(invitation);
+            Assert.IsNotNull(invitation.RowVersion);
+        }
+
+        //[TestMethod]
```

Comment:
Are we waiting on mock data before uncommenting this?

Reply:
Not mocking but it correspond to the Todo at line 42 in InvitationsControllerTestsHelper.cs. The bodyPayload at line 43 there is not complete yet to run a correct positve test. Get BadRequest due to validation rule(s) for now. You fix